glue/src/it/scala/com/gu/tableversions/glue/GlueMetastoreSpec.scala

```diff
@@ -0,0 +1,115 @@
+package com.gu.tableversions.glue
+
+import java.net.URI
+
+import cats.effect.IO
+import cats.implicits._
+import com.amazonaws.auth.{
+  EnvironmentVariableCredentialsProvider,
+  InstanceProfileCredentialsProvider,
+  SystemPropertiesCredentialsProvider,
+  _
+}
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.services.glue.model._
+import com.amazonaws.services.glue.{AWSGlue, AWSGlueClient}
+import com.gu.tableversions.core.Partition.PartitionColumn
+import com.gu.tableversions.core._
+import com.gu.tableversions.metastore.MetastoreSpec
+import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
+
+import scala.util.{Properties, Random}
+
+class GlueMetastoreSpec extends FlatSpec with Matchers with BeforeAndAfterAll with MetastoreSpec {
+
+  def readMandatoryEnvVariable(varName: String) =
+    Properties.envOrNone(varName).toRight(s"$varName environment variable must be set")
+
+  val AWSProfileEnvVarName = "TABLE_VERSIONS_TEST_AWS_PROFILE"
+  val AWSRegionEnvVarName = "TABLE_VERSIONS_TEST_AWS_REGION"
```

Comment:
I think it should be possible to just load the aws_region in your aws profile configuration but it doesn't seem to work for me so I had to explicitly add the region as another variable here.